### PR TITLE
Port #18875 to Cloud 2.0

### PR DESCRIPTION
**Pull request by mdlinville**
Port #18875 to Cloud 2.0

Strategy: Cherry-pick, then manually resolve conflicts.

### CODE REVIEW

File: src/current/cockroachcloud/alerts-page.md
Comment (rmloveland) [inline src/current/cockroachcloud/alerts-page.md:23]: not sure where this link anchor is coming from, in https://github.com/cockroachdb/docs/pull/18875/files#diff-2e6f522d8fc3cc357a2551b2ad5112c5af53419fb52710f61e40c053bfabbab2R22 the anchors are updated to link to the `#support-phases` anchor which exists (this one does not)

It seems we should merge in the actual changes from #18875 which did not have this build error 

This broken link makes me think this is not a merge, but trying to manually add the edits?

File: src/current/cockroachcloud/alerts-page.md
Comment (rmloveland) [inline src/current/cockroachcloud/alerts-page.md:32]: this whitespace commit suggests this is not a merge but a manual port. is that correct?

doesn't that mean when we merge `cloud-2.0` back into `main` we're going to potentially lose work that was done on `main` that didn't get manually ported over here?

File: src/current/cockroachcloud/connect-to-your-cluster.md
Comment (rmloveland) [inline src/current/cockroachcloud/connect-to-your-cluster.md:182]: this part def looks like a content change, not a merge of the link edits

File: src/current/cockroachcloud/export-logs.md
Comment (rmloveland) [inline src/current/cockroachcloud/export-logs.md:348]: This is introducing a typo: `Clo ud`

AFAICT this typo is not in `main` nor in #18875 

?

(PS maybe i'm missing something here, I haven't worked in this branch in a long time)

File: src/current/cockroachcloud/troubleshooting-page.md
Comment (rmloveland) [inline src/current/cockroachcloud/troubleshooting-page.md:43]: the reordering of standard and basic is not in #18875

**Review by rmloveland — APPROVED** [review]
LGTM once build is green but i feel like i'm missing something here

There appear to be multiple edits in this PR that are not just merging stuff from #18875 ?

that seems strange since it seems like if we can't merge stuff over from main without doing manual edits that's ... not good, right?

that said it looks like all the links are here and you know git better than me, and maybe i'm misunderstanding what is happening here so I defer to your expertise in maintaining this long-running branch

File: src/current/cockroachcloud/alerts-page.md
Comment (mdlinville) [reply src/current/cockroachcloud/alerts-page.md]: We will have conflicts to resolve, we won't lose work. I cherry-picked your PR's commit before you merged it, resolved Cloud-2.0-specific conflicts, then poked around looking for other fully-qualified links that were in `cloud-2.0` only. Sometimes I opened a file in my editor to make an edit, and then whitespace may have gotten trimmed. Some areas your PR changed had already been previously edited in `cloud-2.0` also, so the contents here are different from the ones in `main`.

File: src/current/cockroachcloud/troubleshooting-page.md
Comment (mdlinville) [reply src/current/cockroachcloud/troubleshooting-page.md:43]: Correct, I flipped them on purpose because we are supposed to highlight Standard before Basic.

**Pull request merged**
